pkg/harvester/edit/harvesterhci.io.host/HarvesterSeeder.vue

```diff
@@ -161,16 +192,17 @@ export default {
       this.hide();
     },
 
-    resetFields() {
-      this.secret.data.username = '';
-      this.secret.data.password = '';
-      this.secret.metadata.name = '';
-      this.secret.metadata.namespace = '';
-    },
-
     async saveSecret(buttonCb) {
       this.errors = [];
 
+      if (!this.username) {
+        this.errors.push(this.t('validation.required', { key: this.t('harvester.virtualMachine.input.username') }, true));
+      }
+
+      if (!this.password) {
+        this.errors.push(this.t('validation.required', { key: this.t('harvester.virtualMachine.input.password') }, true));
+      }
```

Comment:
There is a problem here, when we create a secret and it is not used, if the user does not refresh the page, the secret we have just created will be displayed in the secret select.   https://github.com/harvester/dashboard/pull/695/files#diff-79df5862d6fc8da90bf9ef46c59b48de3624f912681c36583c6ac7bee00e37a7L190

Reply:
https://user-images.githubusercontent.com/24985926/236984821-bb8db8d4-eafb-450c-b243-755f0e504e32.mov